src/components/pages/CommunityPage/CommunityPage.tsx

```diff
@@ -64,7 +64,10 @@ export default function CommunityPage(): JSX.Element {
   });
 
   function MemberCard({ member, index }) {
-    const githubUsername = member.githubUsername?.[0] || member.apacheId;
+    // require the member to have "GitHub username(s) (user-provided)" field information in Whimsy
+    // the user can go to Whimsy url https://whimsy.apache.org/roster/committer/__self__ to update the information
```

Comment:
I just tried, but it looks broken:

<img width="529" alt="Screenshot 2024-03-19 at 4 44 15 PM" src="https://github.com/apache/pulsar-site/assets/9302460/dce8d71b-f028-4c57-9128-58c4aa87f6b2">

At the same time after I clicked the "Link GitHub username(s) to ASF id":

<img width="687" alt="Screenshot 2024-03-19 at 4 45 26 PM" src="https://github.com/apache/pulsar-site/assets/9302460/c051b4c2-62dd-4a3d-b7d0-e62976f2fa82">

Reply:
The UI is very confusing. You should be able to **double click** on the "GitHub username(s) (user-provided)" **label** to enter your GitHub Id.

<img width="476" alt="image" src="https://github.com/apache/pulsar-site/assets/66864/6303b170-977e-40cc-be91-89a2872b4d92">

Reply:
Although no, I changed my mind)
Then spend time recovering

<img width="453" alt="Screenshot 2024-03-19 at 4 49 26 PM" src="https://github.com/apache/pulsar-site/assets/9302460/e2cab8b0-afe9-4bfa-abab-9c5746ac4494">

Reply:
Looks fine now, I updated team.js.